src/app/submit-item/components/submit/submit.component.ts

```diff
@@ -114,6 +120,11 @@ export class SubmitComponent implements OnInit {
     this.submitEnabled = true;
   }
 
+  disabled(){
+    const result = !this.submitEnabled || !this.privacyBox.value || !this.termBox.value;
+    return result;
```

Comment:
In my opinion you don't need to declare a constant that you simply return in the line below.
I'd rather do:

`return !this.submitEnabled || !this.privacyBox.value || !this.termBox.value;`

Not necessary though :)